bundles/org.openhab.binding.velux/README.md

```diff
@@ -110,12 +110,19 @@ Notes:
 1. To enable a complete inversion of all parameter values (i.e. for Velux windows), use the property `inverted` or add a trailing star to the eight-byte serial number.
   For an example, see the Thing definition for 'Bathroom_Roof_Window' below.
```

Comment:
```suggestion
```
See comment above. I would suggest to drop this and update the example as the inverted property is not needed for windows.